projects/storefrontlib/src/cms-components/misc/icon/icon-loader.service.ts

```diff
@@ -12,31 +14,59 @@ import {
 })
 export class IconLoaderService {
   private loadedResources = [];
-  constructor(protected winRef: WindowRef, protected config: IconConfig) {}
+  constructor(
+    protected winRef: WindowRef,
+    protected iconConfig: IconConfig,
+    protected sanitizer: DomSanitizer
+  ) {}
 
   /**
-   * Indicates whether the given icon type is configured to use SVG.
+   * Returns an html fragment which can be added to the DOM in a safe way.
    */
-  useSvg(iconType: ICON_TYPE): boolean {
```

Comment:
Renaming/Deleting method is a breaking change

Reply:
Same case with `useSvg` on component

Reply:
updated the (prepared) commit message